umm-spec-lib/test/cmr/umm_spec/test/xml_to_umm_mappings/iso_shared/distributions_related_url.clj

```diff
@@ -74,6 +74,37 @@
         :Checksum nil,
         :MimeType nil}}))
 
+(def expected-distribution-related-url-record-CMR-5366
```

Comment:
Since commas are not supposed to be used to separate the values can you thrown in valid URLContentType, and Types.

Reply:
I made a code suggestion in the CMR_5366.xml file of what I mean.